benchmarks/BenchmarkHelper.jl

```diff
@@ -1,129 +1,21 @@
-    branch = replace(CURRENT_BRANCH[], r"[-/.]"=>"_")
-    outdir= joinpath(PROJECT_DIR, "benchmarks", "output")
-    mkpath(outdir)
-    return joinpath(outdir, "BM-$(CURRENT_JOB_ID[])-$branch.txt")
-end
-
-function run_suite(tags_expr=:ALL)
-    benchmarks = eval(Meta.parse("BenchmarkSuite[@tagged $tags_expr]"))
-    t = run(benchmarks, samples=1)
-    branch = CURRENT_BRANCH[]
-    @info "benchmark result", t
-    open(output_path(), "w") do io
-        show(io, bmresults_to_dataframe(t))
-    end
-end
-
 # Functions run in the dispather process
 
-function gitcurrentbranch()
-    branches = readlines(`git branch`)
-    ind = findfirst(x -> x[1] == '*', branches)
-    brname = branches[ind][3:end]
-    if brname[1] == '('
-        brname = readlines(`git rev-parse --short HEAD`)[1]
-    end
-    return brname
-end
-
-function command_info(event_data)
-    comment_body = event_data["comment"]["body"]
-    !occursin("@BayesBot", comment_body) && return (nothing, nothing, nothing)
-    reg_bm_cmd = r"`\w+\(.*\)`"i
-    bm_cmd = match(reg_bm_cmd, comment_body)
-    bm_cmd == nothing && return (nothing, nothing, nothing)
-    cmd, args, kwargs = parse_submission_string(bm_cmd.match)
-    return (cmd, args, kwargs)
-end
-
-function benchmarks_info(event_data)
-    cmd, args, kwargs = command_info(event_data)
-    if cmd != "runbenchmarks"
-        return false, nothing, nothing
-    end
-
-    branches =  haskey(kwargs, :vs) ? Meta.parse(kwargs[:vs]) : []
-    if branches isa Expr && branches.head == :vect
-        branches = branches.args
-    elseif branches isa AbstractString
-        branches = [branches]
-    end
-    target = "master"
-    if haskey(event_data, "pull_request")
-        target = event_data["pull_request"]["head"]["ref"]
-    end
-    if isempty(branches)
-        if target != "master"
-            branches = [target, "master"]
-        end
-    elseif length(branches) == 1
-        if !in(target, branches)
-            push!(branches, target)
-        elseif !in("master", branches)
-            push!(branches, "master")
-        end
-    end
-    return true, Meta.parse(args[1]), branches
-end
-
-function target_benchmarks(bm_text)
-    return filter(!isempty, map(strip, split(bm_text, ",")))
-end
-
-function reply_comment(event_data, body)
-    repo = event_data["repository"]["full_name"]
-    # auth = GitHub.authenticate(get(ENV, "GITHUB_TOKEN", ""))
-    auth = GitHub.authenticate(get(ENV, "BENCHMARK_KEY", ""))
-    params = Dict("body" => body)
-
-    comment_kind = :issue
-    comment_parent = nothing
-
-    if haskey(event_data, "issue")
-        comment_kind = :issue
-        comment_parent = event_data["issue"]["number"]
-    elseif haskey(event_data["comment"], "commit_id")
-        comment_kind = :commit
-        comment_parent = event_data["comment"]["commit_id"]
-    end
-
-    create_comment(repo, comment_parent, comment_kind;
-                   params=params, auth=auth)
-end
-
 function benchmark_excluded(file)
```

Comment:
Do we still need this, now that we can use `tags` from BenchmarkTools to select benchmarks?